const getProduct = async () => {
    try {
const response = await fetch();
    } catch (error) {
        console.error(error);
    }
}